components/Tabs/src/tabs.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 
 interface TabsProps {
-  items: Array<Partial<Record<"name" | "active" | "id", any>>>;
+  items: Array<Partial<Record<"name" | "active" | "id" | "onClick", any>>>;
```

Comment:
I think this `Array<Partial<Record...` is hard to understand an hard to maintain. Something like this might work better:

```
tabs: {
  name: string,
  active: boolean,
  id: string,
  onClick?: () => void
}[]
```

Reply:
Right now, we can push anything we want into the `onClick` prop. E.g. `onClick=1` would be a valid prop to send.

This would render into an `li` that has an `onClick` set to `1`, which would throw an error.